tests/Console/ConsoleEventSchedulerTest.php

```diff
@@ -24,20 +27,22 @@ public function testExecCreatesNewCommand()
         $this->assertEquals('path/to/command', $events[0]->command);
         $this->assertEquals('path/to/command -f --foo="bar"', $events[1]->command);
         $this->assertEquals('path/to/command -f', $events[2]->command);
-        $this->assertEquals('path/to/command --foo=\'bar\'', $events[3]->command);
-        $this->assertEquals('path/to/command -f --foo=\'bar\'', $events[4]->command);
-        $this->assertEquals('path/to/command --title=\'A "real" test\'', $events[5]->command);
+        $this->assertEquals('path/to/command --foo='.$escape.'bar'.$escape, $events[3]->command);
+        $this->assertEquals('path/to/command -f --foo='.$escape.'bar'.$escape, $events[4]->command);
```

Comment:
@GrahamCampbell 

It's not possible because Windows accept only double quotes, and Unix uses the `escapeshellarg()`, that will use single quotes. So Symfony reproduced that on `symfony/process` at `ProcessUtils.php:39`.

Seems that we have two PHP bugs related to `escapeshellarg()` on Windows:
- https://bugs.php.net/bug.php?id=43784
- https://bugs.php.net/bug.php?id=49446

An alternative solution is replace single quotes to double quotes on tests, but it'll do exactly the same that I do, but will be even more like a workaround.